### ISSUE TITLE
Missing type file in dist folder

### ISSUE THREAD

**Issue by Ploud42** [Noction/vue-bezier#4]
Hi,

Since I updated to 1.0.9 I'm getting this error

```
ERROR(vue-tsc)  Could not find a declaration file for module '@noction/vue-bezier'. '[...]/Development/VueJS/manager-v2/node_modules/@noction/vue-bezier/dist/vue-bezier.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/noction__vue-bezier` if it exists or add a new declaration (.d.ts) file containing `declare module '@noction/vue-bezier';`
```
 
 It seems like the type file is missing from dist folder

![Capture d’écran 2024-01-08 à 10 26 25](https://github.com/Noction/vue-bezier/assets/38037024/937a5d94-0eb7-4518-a98a-cb7eaebd7dcd)

**Comment by 50rayn** [comment]
Hello @Ploud42 ,
Types are now available from v1.10.0.

<img width="217" alt="image" src="https://github.com/Noction/vue-bezier/assets/27418268/df568c1e-53fc-4961-8a39-192aa3231a6d">

Thanks for the report.

**Comment by glutaminefree** [comment]
Still getting this error

![image](https://github.com/Noction/vue-bezier/assets/2122419/87cab2cc-317d-4ced-91c4-b5a7e2e9b800)


```
Could not find a declaration file for module '@noction/vue-bezier'. 'path-to-project/node_modules/@noction/vue-bezier/dist/vue-bezier.js' implicitly has an 'any' type.
  There are types at 'path-to-project/node_modules/@noction/vue-bezier/dist/types/main.d.ts', but this result could not be resolved when respecting package.json "exports". The '@noction/vue-bezier' library may need to update its package.json or typings.ts(7016)
  ```
